nvflare/security/logging.py

```diff
@@ -130,4 +130,4 @@ def secure_format_exception(e: Exception) -> str:
     if is_secure():
         return str(type(e))
     else:
-        return str(e)
+        return f"{str(type(e))}: {str(e)}"
```

Comment:
I've recently noticed this doesn't print what I expected, and should be fixed thus:
```suggestion
        return f"{type(e).__name__}: {str(e)}"
```

Reply:
thanks, I will change accordingly